src/test/unit/storage/test_db_interface_common.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from storage.db_interface_common import MongoInterfaceCommon
+
+
+class CommonInterfaceMock(MongoInterfaceCommon):
+
+    def __init__(self):
+        pass
+
+    def retrieve_analysis(self, sanitized_dict, analysis_filter=None):
+        return {}
+
+
+current_data_format = {
+    '_id': 'some_UID',
+    'size': 1,
+    'file_name': 'name_of_the_file',
+    'device_name': 'test_device',
+    'device_class': 'class_of_the_device',
+    'release_date': 0,
+    'vendor': 'test_vendor',
+    'version': '0.1',
+    'processed_analysis': {},
+    'files_included': [],
+    'virtual_file_path': {},
+    'tags': {},
+    'analysis_tags': {},
+    'device_part': 'bootloader'
+}
+
+old_data_format = {
+    '_id': 'some_UID',
+    'size': 1,
+    'file_name': 'name_of_the_file',
+    'device_name': 'test_device',
+    'device_class': 'class_of_the_device',
+    'release_date': 0,
+    'vendor': 'test_vendor',
+    'version': '0.1',
+    'processed_analysis': {},
+    'files_included': [],
+    'virtual_file_path': {},
+    'comment': 'some comment'
```

Comment:
Where are comments in the new data format?

Reply:
comments are stored in another collection in the new format